core/controllers/feedback.py

```diff
@@ -263,3 +264,21 @@ def post(self):
             feedback_services.clear_feedback_message_references, self.user_id,
             exploration_id, thread_id)
         self.render_json(self.values)
+
+
+class UnsentSuggestionEmailHandler(base.BaseHandler):
```

Comment:
I think just SuggestionEmailHandler is fine, since we don't really have a concept here of "unsent" suggestion emails.